source/components/organisms/FormList/FormList.js

```diff
@@ -15,13 +15,13 @@ const ListHeading = styled(Heading)`
 `;
 
 const FormList = ({ onClickCallback, heading, showSubforms }) => {
-  const [forms, setForms] = useState([]);
-  const { getFormSummaries } = useContext(FormContext);
+  const [formSummaries, setFormSummaries] = useState([]);
+  const { getFormSummaries, getForm } = useContext(FormContext);
 
   useEffect(() => {
     async function fetchForms() {
       const formSummaries = await getFormSummaries();
-      setForms(formSummaries.filter(f => (showSubforms ? f.subform : !f.subform)));
+      setFormSummaries(formSummaries.filter(f => (showSubforms ? f.subform : !f.subform)));
     }
     fetchForms();
```

Comment:
Are there any reason for fetching formSummaries in this component instead of in the context and then retreive the summaries from context? 
From what i can see we are setting an identical state for formSummaries in from context.